credsweeper/common/keyword_checklist.txt

```diff
@@ -942,4 +942,4 @@ yellow
 yield
 zeros
 .json
-.xml
\ No newline at end of file
+.xml
```

Comment:
Maybe I'm blind but I cannot see where is a difference :D
I assume it's related to a newline but git not showing it